document/core/syntax/modules.rst

```diff
@@ -213,12 +207,6 @@ Memories are referenced through :ref:`memory indices <syntax-memidx>`,
 starting with the smallest index not referencing a memory :ref:`import <syntax-import>`.
 Most constructs implicitly reference memory index :math:`0`.
 
-.. note::
-   In the current version of WebAssembly, at most one memory may be defined or imported in a single module,
-   and *all* constructs implicitly reference this memory :math:`0`.
```

Comment:
I think this one still holds, until the multi-memory proposal is accepted.

Reply:
Fixed, thanks.